main.go

```diff
@@ -70,6 +75,11 @@ func main() {
 								Name:  "all",
 								Usage: "List all projects",
 							},
+							&cli.StringFlag{
+								Name:        "output",
```

Comment:
Love it, one improvement I could see was to provide a list of options to output with the help flag ("json", "table") but it doesn't look like you can do that with the flag package, is that right?

Reply:
Yeah doesnt provide a way to have a fixed list of options. You can do custom validation for flag values, but seems a bit overkill.

I've updated the help text to include table and json